src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -18,13 +19,15 @@ export default class SimidMapCreative extends BaseSimidCreative {
 
   constructor() {
     super();
+    this.newUserSession = new UserActivityLogger();
```

Comment:
Add type params, and make private (private properties end in `_`)

Reply:
added